.github/workflows/future-effective-filings-ci.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         run: |
           make test
       #- name: Upload coverage to Codecov
-      #  uses: codecov/codecov-action@v1
```

Comment:
without knowing much about codecov-action API, maybe the commented out sections can be removed, as they were relevant to v1 (I assume they were commented out during migration to v2, which appears to have a different pattern)

Reply:
We'll still need it when test cased got fixed.